spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -95,8 +97,8 @@ public static Object[][] parameters() {
   @TempDir private Path temp;
 
   @BeforeEach
-  public void setupTempDirs() {
-    fileTableDir = temp.toFile();
+  public void setupTempDirs() throws IOException {
+    fileTableDir = Files.createTempDirectory(temp, "junit").toFile();
```

Comment:
`temp` is already a temporary directory, so we don't need to do this here

Reply:
Okay sure. will revert this part.